Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SurfaceMagnetism.cs

```diff
@@ -77,28 +77,28 @@ public LayerMask[] MagneticSurfaces
 
         [SerializeField]
         [Tooltip("Max distance for raycast to check for surfaces")]
-        private float maxDistance = 3.0f;
+        private float maxRaycastDistance = 50.0f;
 
         /// <summary>
         /// Max distance for raycast to check for surfaces
         /// </summary>
-        public float MaxDistance
+        public float MaxRaycastDistance
```

Comment:
>MaxRaycastDistance [](start = 21, length = 18)

Please add a note on Solver migration in GA here:

[https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/UpdatingToGA.html](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/UpdatingToGA.html)